---
layout: default
title: Papers
parent: Readings
nav_order: 1
---

Lectures and activities in this course will draw from the following collection
of readings. The readings draw from the statistics, machine learning, and
computational molecular biology communities. We have included both modern
references and classic papers in genomics data analysis.  Based on student
interests, we will choose 1 - 3 papers a week.

<details open markdown="block">
  <summary>
    Table of contents
  </summary>
  {: .text-delta }
1. TOC
{:toc}
</details>

## Scientific Context

1. [Excerpt from "The Song of the Cell"](https://news.harvard.edu/gazette/story/2022/11/excerpt-from-siddhartha-mukherjees-the-song-of-the-cell/)
1.	[Computational principles and challenges in single-cell data integration](https://doi.org/10.1038/s41587-021-00895-7)
1.	[Integrative single-cell analysis](https://doi.org/10.1038/s41576-019-0093-7)
1.	[Statistics requantitates the central dogma](https://www.science.org/doi/10.1126/science.aaa8332)

## Data Management

###	Data Structures

1.	[Muon: multimodal omics analysis framework](https://doi.org/10.1186/s13059-021-02577-8)
1.	[Software for the Integration of Multiomics Experiments in Bioconductor](http://cancerres.aacrjournals.org/lookup/doi/10.1158/0008-5472.CAN-17-0344)

### Processing and Visualization
1. [Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R](https://doi.org/10.1093/bioinformatics/btw777)
1. [Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq](https://linkinghub.elsevier.com/retrieve/pii/S2405471219300808)
1. [Fast, sensitive and accurate integration of single-cell data with Harmony](https://doi.org/10.1038/s41592-019-0619-0)
1. [Evaluation of statistical methods for normalization and differential expression in mRNA-Seq experiments](https://doi.org/10.1186/1471-2105-11-94)

## Hypothesis Testing

### Overviews

1. [Statistical methods for identifying differentially expressed genes in replicated cDNA microarray experiments](https://www.jstor.org/stable/pdf/24307038.pdf)
1. [A practical guide to methods controlling false discoveries in computational biology](https://doi.org/10.1186/s13059-019-1716-1)

### Proposals

1.	[Clipper: p-value-free FDR control on high-throughput data from two conditions](https://doi.org/10.1186/s13059-021-02506-9)
1. [Exaggerated false positives by popular differential expression methods when analyzing human population samples](https://doi.org/10.1186/s13059-022-02648-4)
1. [High-sensitivity pattern discovery in large, paired multiomic datasets](https://doi.org/10.1093/bioinformatics/btac232)
1.	[A selective inference approach for false discovery rate control using multiomics covariates yields insights into disease risk](https://www.pnas.org/content/117/26/15028)
1.	[A statistical approach for identifying differential distributions in single-cell RNA-seq experiments](https://doi.org/10.1186/s13059-016-1077-y)

## Prediction and Dimensionality Reduction

### Unsupervised

1.	[Bayesian group factor analysis with structured sparsity](https://jmlr.org/papers/v17/14-472.html)
1.	[Dimension reduction techniques for the integrative analysis of multi-omics data](https://academic.oup.com/bib/article-lookup/doi/10.1093/bib/bbv108)
1.	[Multi-Omics Factor Analysis -- framework for unsupervised integration of multi-omics data sets](https://doi.org/10.15252/msb.20178124)
1.	[A Generic Multivariate Framework for the Integration of Microbiome Longitudinal Studies with Other Data Types](https://www.frontiersin.org/article/10.3389/fgene.2019.00963)
1. [Jointly Embedding Multiple Single-Cell Omics Measurements](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8496402/#:~:text=10.4230/LIPIcs.WABI.2019.10)
1. [Joint and individual variation explained (JIVE) for integrated analysis of multiple data types](https://projecteuclid.org/journals/annals-of-applied-statistics/volume-7/issue-1/Joint-and-individual-variation-explained-JIVE-for-integrated-analysis-of/10.1214/12-AOAS597.full)
1. [Integrated principal components analysis](https://dl.acm.org/doi/abs/10.5555/3546258.3546456)
1. [Integrative analysis of single-cell genomics data by coupled nonnegative matrix factorizations](https://www.pnas.org/doi/10.1073/pnas.1805681115)
1.	[Alignment of spatial genomics and histology data using deep Gaussian processes](https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2022.01.10.475692.full.pdf)
1.	[Inference after latent variable estimation for single-cell RNA sequencing data](https://doi.org/10.1093/biostatistics/kxac047)

###	Supervised

1.	[Sparse partial least squares regression for simultaneous dimension reduction and variable selection](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1467-9868.2009.00723.x)
1. [Feature selection for data integration with mixed multiview data](https://doi.org/10.1214/20-AOAS1389)
1. [Cooperative learning for multiview analysis](https://www.pnas.org/doi/10.1073/pnas.2202113119)
1. [Efficient multivariate linear mixed model algorithms for genome-wide association studies](https://doi.org/10.1038/nmeth.2848)
1. [mixOmics: An R package for 'omics feature selection and multiple data integration](https://doi.org/10.1371/journal.pcbi.1005752)
1. [Joint association and classification analysis of multi-view data](https://arxiv.org/abs/1811.08511)
1. [Multimodal Single-Cell Translation and Alignment with Semi-Supervised Learning](https://www.liebertpub.com/doi/pdf/10.1089/cmb.2022.0264)
1.	[Large-Margin Predictive Latent Subspace Learning for Multiview Data Analysis](http://ieeexplore.ieee.org/document/6332444/)

## Generative Modeling

1.	[Simultaneous deep generative modelling and clustering of single-cell genomic data](https://www.nature.com/articles/s42256-021-00333-y)
1. [Joint probabilistic modeling of single-cell multi-omic data with totalVI](http://www.nature.com/articles/s41592-020-01050-x)
5.	[Model-Based Approach to the Joint Analysis of Single-Cell Data on Chromatin Accessibility and Gene Expression](https://projecteuclid.org/journals/statistical-science/volume-35/issue-1/Model-Based-Approach-to-the-Joint-Analysis-of-Single-Cell/10.1214/19-STS714.full)
2.	[Spatially Aware Dimension Reduction for Spatial Transcriptomics](https://doi.org/10.1038/s41467-022-34879-1)
3.	[Dynamic Bayesian Networks for Integrating Multi-omics Time Series Microbiome Data](https://journals.asm.org/doi/10.1128/mSystems.01105-20)
4.	[Identifying temporal and spatial patterns of variation from multimodal data using MEFISTO](https://www.nature.com/articles/s41592-021-01343-9)
6.	[Robust probabilistic modeling for single-cell multimodal mosaic integration and imputation via scVAEIT](https://doi.org/10.1073/pnas.2214414119)
1. [FreeHi-C spike-in simulations for benchmarking differential chromatin interaction detection](https://www.sciencedirect.com/science/article/pii/S1046202320300852)
1. [Multimodal Learning with Deep Boltzmann Machines](https://proceedings.neurips.cc/paper/2012/hash/af21d0c97db2e27e13572cbf59eb343d-Abstract.html)

## Networks

1.	[Inferring Interaction Networks From Multi-Omics Data](https://www.frontiersin.org/articles/10.3389/fgene.2019.00535)
2.	[Network modeling in biology: statistical methods for gene and brain networks](https://doi.org/10.1214/20-STS792)
3.	[Computational inference of gene regulatory networks: Approaches, limitations and opportunities](https://linkinghub.elsevier.com/retrieve/pii/S1874939916301882)
4.	[Interpretation of network-based integration from multi-omics longitudinal data](https://doi.org/10.1093/nar/gkab1200)
5.	[Network Modeling of Topological Domains using HI-C Data](https://doi.org/10.1214/19-AOAS1244)
6.	[Integrative Spatial Single-cell Analysis with Graph-based Feature Learning](https://doi.org/10.1101/2020.08.12.248971)

## Multi-Study Analysis

###	Meta-Analysis
1. [Multiple-laboratory comparison of microarray platforms](https://doi.org/10.1038/nmeth756)
1. [Meta-analysis methods for multiple related markers: Applications to microbiome studies with the results on multiple α‐diversity indices](https://onlinelibrary.wiley.com/doi/10.1002/sim.8940)
1. [Meta-analysis of gut microbiome studies identifies disease-specific and shared responses](https://doi.org/10.1038/s41467-017-01973-8)
1. [The impact of different sources of heterogeneity on loss of accuracy from genomic prediction models](https://academic.oup.com/biostatistics/article/21/2/253/5092386)
1. [Accessible, curated metagenomic data through ExperimentHub](https://doi.org/10.1038/nmeth.4468)

### Causality, Bayes, and Transfer Learning
1. [A Unified Framework for Association Analysis with Multiple Related Phenotypes](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0065245)
1. [Anchor regression: Heterogeneous data meet causality]( https://doi.org/10.1111/rssb.12398)
1. [Data denoising with transfer learning in single-cell transcriptomics](http://www.nature.com/articles/s41592-019-0537-1)
1. [scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning](https://www.nature.com/articles/s41587-021-01161-6)
1. [Integration and transfer learning of single-cell transcriptomes via cFIT](https://pnas.org/doi/full/10.1073/pnas.2024383118)
1. [INFIMA leverages multi-omics model organism data to identify effector genes of human GWAS variants](https://doi.org/10.1186/s13059-021-02450-8)
