---
layout: default
title: Reading List
nav_order: 2
---

# Readings

## Background

1.	[Genomics and Systems Biology](https://drive.google.com/file/d/1FJQMsQySjTA2KK1JQU73PB4OvjM_v2n7/view?usp=share_link)
1.	[Statistics requantitates the central dogma](https://www.science.org/doi/10.1126/science.aaa8332)
1.	[Computational principles and challenges in single-cell data integration](https://doi.org/10.1038/s41587-021-00895-7)
1.	[Integrative single-cell analysis](https://doi.org/10.1038/s41576-019-0093-7)
1.	[The integrative human microbiome project](https://doi.org/10.1038/s41586-019-1238-8)

## Data Management, Processing, and Visualization

###	Data Structures

1.	[Muon: multimodal omics analysis framework](https://doi.org/10.1186/s13059-021-02577-8)
1.	[Software for the Integration of Multiomics Experiments in Bioconductor](http://cancerres.aacrjournals.org/lookup/doi/10.1158/0008-5472.CAN-17-0344)

### Processing and Visualization
1. [Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R](https://doi.org/10.1093/bioinformatics/btw777)
1. [Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq](https://linkinghub.elsevier.com/retrieve/pii/S2405471219300808)
1. [Fast, sensitive and accurate integration of single-cell data with Harmony](https://doi.org/10.1038/s41592-019-0619-0)
1. [Evaluation of statistical methods for normalization and differential expression in mRNA-Seq experiments](https://doi.org/10.1186/1471-2105-11-94)

## Hypothesis Testing

### Overviews

1. [Resampling-based multiple testing for microarray data analysis](https://doi.org/10.1007/BF02595811)
1. [A practical guide to methods controlling false discoveries in computational biology](https://doi.org/10.1186/s13059-019-1716-1)

### Proposals

1.	Clipper: p-value-free FDR control on high-throughput data from two conditions
1.	A selective inference approach for false discovery rate control using multiomics covariates yields insights into disease risk
1.	A statistical approach for identifying differential distributions in single-cell RNA-seq experiments

## Prediction and Dimensionality Reduction

###	Statistical

1.	Sparse partial least squares regression for simultaneous dimension reduction and variable selection
1.	Bayesian group factor analysis with structured sparsity
1.	Large-Margin Predictive Latent Subspace Learning for Multiview Data Analysis

### Dimensionality Reduction

1.	Dimension reduction techniques for the integrative analysis of multi-omics data
1.	Multi-Omics Factor Analysis—a framework for unsupervised integration of multi-omics data sets
1.	A Generic Multivariate Framework for the Integration of Microbiome Longitudinal Studies with Other Data Types
1.	Inference after latent variable estimation for single-cell RNA sequencing data
1.	Nonnegative spatial factorization

###	Prediction

1.	A selective review of multi-level omics data integration using variable selection
1.	Cooperative learning for multiview analysis
1.	mixOmics: An R package for 'omics feature selection and multiple data integration

## Generative Modeling

1.	Simultaneous deep generative modelling and clustering of single-cell genomic data
2.	Spatially Aware Dimension Reduction for Spatial Transcriptomics
3.	Dynamic Bayesian Networks for Integrating Multi-omics Time Series Microbiome Data
4.	Identifying temporal and spatial patterns of variation from multimodal data using MEFISTO
5.	Model-Based Approach to the Joint Analysis of Single-Cell Data on Chromatin Accessibility and Gene Expression
6.	Robust probabilistic modeling for single-cell multimodal mosaic integration and imputation via scVAEIT

## Networks

1.	Inferring Interaction Networks From Multi-Omics Data
2.	Network modeling in biology: statistical methods for gene and brain networks
3.	Computational inference of gene regulatory networks: Approaches, limitations and opportunities
4.	Interpretation of network-based integration from multi-omics longitudinal data
5.	Network Modeling of Topological Domains using HI-C Data
6.	Integrative Spatial Single-cell Analysis with Graph-based Feature Learning

## Multi-Study Analysis

###	Empirical Bayes

1.	A Unified Framework for Association Analysis with Multiple Related Phenotypes
1.	Data denoising with transfer learning in single-cell transcriptomics

###	Meta-Analysis
1. Multiple-laboratory comparison of microarray platforms
1.	Meta‐analysis methods for multiple related markers: Applications to microbiome studies with the results on multiple α‐diversity indices
1. Meta-analysis of gut microbiome studies identifies disease-specific and shared responses
1.	The impact of different sources of heterogeneity on loss of accuracy from genomic prediction models
1. Accessible, curated metagenomic data through ExperimentHub

### Transfer Learning
1.	scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning
1.	Integration and transfer learning of single-cell transcriptomes via cFIT

# Adding to the Reading List

You are welcome (encouraged!) to suggest additional readings for this course. 

## Journals

1.	Genome Biology
2.	Nature Methods
3.	Biostatistics
4.	Annals of Applied Statistics
5.	Bioinformatics
6.	BMC Bioinformatics

## Conferences

1.	Bioconductor Conference
2.	Pacific Symposium for Biocomputing
3.	RECOMB Systems Biology

## Miscellaneous

1.	[awesome-multi-omics](https://github.com/mikelove/awesome-multi-omics)
2.	[community-bioc](https://slack.bioconductor.org/)
